src/analyze/analyze-compare-versions.c

```diff
@@ -10,7 +10,7 @@
 int verb_compare_versions(int argc, char *argv[], void *userdata) {
         int r;
 
-        assert(argc == 3 || argc == 4);
+        assert(IN_SET(argc, 3, 4));
```

Comment:
The commit message should be updated, that is, please drop "udev".

Reply:
Whoops, thanks. I'll leave the Cocci-tweak commit for a separate PR, since I need to dig a bit deeper on that one.

Reply:
Nevermind, I think I got it, PTAL.